### ISSUE TITLE
can not run scrapyd on 10.11.5 El Cap.

### ISSUE THREAD

**Issue by yilmazalican** [scrapy/scrapyd#184]
I've installed scrapyd with pip. Unexpectly i'm getting error when i'm running "scrapyd" command on terminal.

```
Alican:~ alicanyilmaz$ scrapyd
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.5/bin/scrapyd", line 9, in <module>
    load_entry_point('scrapyd==1.1.0', 'console_scripts', 'scrapyd')()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pkg_resources/__init__.py", line 547, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2720, in load_entry_point
    return ep.load()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2380, in load
    return self.resolve()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2386, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/scrapyd-1.1.0-py3.5.egg/scrapyd/__init__.py", line 3, in <module>
    version_info = tuple(__version__.split('.')[:3])
**TypeError: a bytes-like object is required, not 'str'**
```

**Comment by Digenis** [comment]
You are using python3, scrapyd doesn't support it yet.
The python3 port of scrapyd is in progress.

**Comment by manukankani** [comment]
Is scrapy community really active? 
Its been long time. I don't see any effort to make it work for python 3.5

**Comment by Digenis** [comment]
scrapyd-1.2.0a1 is released today.
This experimental release includes full python3 support.
You can try it out with `pip install scrapyd'==1.2.0a1'`
Read the full announcement  https://groups.google.com/forum/#!topic/scrapy-users/6xSS98svwe4

@yilmazalican, if you encounter any problems specific to MacOS,
don't hesitate to report them here.
Although scrapyd is not tested enough in MacOS,
we can probably work towards full support in the upcoming 1.2

**Comment by Digenis** [comment]
@yilmazalican, did you have a chance to try the alpha release in MacOS?

**Comment by Digenis** [comment]
This issue is resolved in the new release, scrapyd-1.2
See #143
